IBAnimatable/AnimatablePresentatedViewController.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  Created by Tom Baranes on 16/07/16.
+//  Copyright © 2016 Jake Lin. All rights reserved.
+//
+
+import UIKit
+
+public class AnimatablePresentedViewController: UIViewController, PresentationDesignable {
```

Comment:
I think `AnimatableModalViewController` maybe an optional because Apple use  [UIModalPresentationStyle](https://developer.apple.com/library/ios/documentation/UIKit/Reference/UIViewController_Class/#//apple_ref/c/tdef/UIModalPresentationStyle), it consist of some options like `PageSheet`, `FullScreen` and `Popover`, another idea is `AnimatablePresenterViewController`. I see Apple use `Presented` as input parameters only but not as a class name.  `PresentedViewController` means a running status of a ViewController. Have you find any class name with `Presented`?